sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 * Added new properties to PeekedMessaged and ReceivedMessage: `enqueued_sequence_number`, `dead_letter_error_description`,
 `dead_letter_reason`, `dead_letter_source`, `delivery_count` and `expires_at_utc`. Please refer to the docstring for further information.
 * Added support for sending received messages via `ServiceBusSender.send_messages`.
+* Added `on_lock_renew_failure` as a parameter to `AutoLockRenew.register`, taking a callback for when the lock is lost non-intentially (e.g. not via settling, shutdown, or autolockrenew duration completion)
+* Added new supported value types int, float, datetime and timedelta for `CorrelationFilter.properties`.
+* Added new property `parameters` to `SqlRuleFilter` and `SqlRuleAction`
```

Comment:
`requires_preprocessing` is also new to SqlRuleAction compared to b4?

Reply:
Yes. Updated.